test/integration/targets/xml/tasks/main.yml

```diff
@@ -67,3 +67,6 @@
   - include_tasks: test-get-element-content.yml
   - include_tasks: test-set-children-elements-unicode.yml
   - include_tasks: test-set-element-value-unicode.yml
+
+  # Filter tests
+  - include_tasks: test-xml-filter.yml
```

Comment:
Put the filter tests in the `filters` target. You'll need to update the requirements list for the test as well:

https://github.com/ansible/ansible/blob/4bd60c313bac585ca012d9729fac8e65da4b2c67/test/integration/targets/filters/runme.sh#L7